### PR TITLE
feat: support webp

### LINKED ISSUE #50: Add WebP output support
WebP offers the same quality and also animations, but needs way less data. (One GIF of mine was 171 KB and the same as WebP was just 13 KB)

### PR DESCRIPTION
**Pull request by caarlos0**
closes #50

need to figure out how to loop webp, tried `-loop -1` and it broke havok, will look more into it tomorrow...

### CODE REVIEW

File: video.go [hunk 1]
```diff
@@ -136,6 +141,40 @@ func MakeWebM(opts VideoOptions) *exec.Cmd {
...
+		fmt.Sprintf(`[0][1]overlay,scale=%d:%d:force_original_aspect_ratio=1,fps=%d,setpts=PTS/%f,pad=%d:%d:(ow-iw)/2:(oh-ih)/2:%s,fillborders=left=%d:right=%d:top=%d:bottom=%d:mode=fixed:color=%s`,
+			opts.Width-2*opts.Padding, opts.Height-2*opts.Padding,
+			opts.Framerate, opts.PlaybackSpeed,
+			opts.Width, opts.Height,
+			opts.BackgroundColor,
+			opts.Padding, opts.Padding, opts.Padding, opts.Padding,
+			opts.BackgroundColor,
+		),
+		"-vcodec", "libwebp",
+		"-pix_fmt", "yuv420p",
```
Comment (the8472) [inline video.go:169]: 4:2:0 chroma subsampling can be bad for sharp, high-contrast edges such as fonts.
There's a lossless version of webp, maybe that supports 4:4:4 chroma?

**Comment by pauloxnet** [comment]
Is this PR blocked by something. How can we help to merge it

**Comment by balupton** [comment]
@pauloxnet seems like it stalled on figuring out how to have ffmpeg generate a looping webp file

**Comment by maaslalani** [comment]
Going to close this for now to clean up some old PRs, we should revisit at some point!

**Pull request closed without merging**